lib/engine/game/base.rb

```diff
@@ -905,6 +905,20 @@ def add_extra_tile(tile)
         extra_cities.each { |c| @_cities[c.id] = c }
       end
 
+      def share_ability_money_on_par(corporation)
```

Comment:
i wonder if this should be a more generic name like share was bought

Reply:
I am not opposed to changing the function name, but I chose this one because it looks for **shares** provided by **abilities** when the corporation is **parred** so it can compensate the treasury **money**. I don't have a better name suggestion.

Reply:
If the idea was to make the function in general less specific, then **on_par(corporation)** or **after_par(corporation)** would be appropriate.

Reply:
yea let's switch it to one of those

Reply:
I switched it to **after_par**